source/scripts/tests/BatTest.js

```diff
@@ -30,5 +31,13 @@ export default function BatTest() {
         // being spawned in.
     })
 
-    // Put assertions here that test your code!
+    // Had to use JSON.stringify to compare objects
```

Comment:
I wasn't a huge fan either. An alternative I thought about was making a helper function that takes two positions and checks whether `position1.x == position2.x && position1.y == position2.y`. I wasn't sure where would be best to put a helper function like that